PREREQUISITES.md

```diff
@@ -143,8 +143,8 @@ The following instructions download the Intel SGX SDK 2.3 and installs it in
 ```bash
 sudo mkdir -p /opt/intel
 cd /opt/intel
-sudo wget https://download.01.org/intel-sgx/linux-2.3.1/ubuntu18.04/sgx_linux_x64_sdk_2.3.101.46683.bin
-echo "yes" | sudo bash ./sgx_linux_x64_sdk_2.3.101.46683.bin
+sudo wget https://download.01.org/intel-sgx/sgx-linux/2.7.1/distro/ubuntu18.04-server/sgx_linux_x64_sdk_2.7.101.3.bin
+echo "yes" | sudo bash ./sgx_linux_x64_sdk_2.7.101.3.bin
```

Comment:
Were you facing any build issues in docker mode ? or only in non docker mode? Any specific reason why we need to use Ubuntu server version ?

Reply:
It does not build on azure. The keenel dricer is outside of Docler.

Reply:
It does not build on azure. The keenel dricer is outside of Docler.

Reply:
It does not build on azure. The keenel dricer is outside of Docler.

Reply:
We also need to update libsgx-enclave-common version. Also could you update the sgx sdk and libsgx-enclave-common in Dockerfile too.

Reply:
Updated libsgx-enclave-common version and Dockerfile.  Please re-review.

Reply:
Were you able to verify this new driver and SDK in SGX hardware enabled system ?

Reply:
Yes. It works with the new driver and SDK on Azure.